source/installer.py

```diff
@@ -186,8 +185,10 @@ def removeOldLibFiles(destPath, rebootOK=False):
 	for topDir in ("lib", "lib64", "libArm64"):
 		currentLibPath = os.path.join(destPath, topDir, versionInfo.version)
 		for parent, subdirs, files in os.walk(os.path.join(destPath, topDir), topdown=False):
-			if parent == currentLibPath:
-				# Lib dir for current installation. Don't touch this!
+			if os.path.commonpath(
+				[os.path.abspath(parent), os.path.abspath(currentLibPath)],
+			) == os.path.abspath(currentLibPath):
+				# We are in the lib dir for current installation. Don't touch this!
```

Comment:
I believe this is safe as it doesn't change the expected behaviour of `parent == currentLibPath` except make it more permissive